File: src/integration/spring/src/jvmTest/java/community/flock/wirespec/integration/spring/java/application/Application.java
package community.flock.wirespec.integration.spring.java.application;

import community.flock.wirespec.integration.spring.java.configuration.EnableWirespecController;
import community.flock.wirespec.integration.spring.java.configuration.EnableWirespecWebClient;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableWirespecController
@EnableWirespecWebClient
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: src/integration/spring/src/jvmTest/kotlin/community/flock/wirespec/integration/spring/application/Application.kt
package community.flock.wirespec.integration.spring.application

import community.flock.wirespec.integration.spring.configuration.WirespecConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(WirespecConfiguration::class)
open class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}